pages/volunteer/VolunteerInfoDefinitions.js

```diff
@@ -15,85 +15,97 @@
 oppiaFoundationWebsite.constant('VOLUNTEER_INFO', {
   design: [{
     fullName: 'Jordan Movish',
-    profilePictureImageFilename: '/static/images/sample_profile_image.jpeg',
+    profilePictureImageFilename: '/static/images/volunteer/profile_images/' +
```

Comment:
In this specification can we just say Jordan_profile_image.jpg, and have the filename interpolation be automatically done in the common code?

Reply:
Are you looking for a constant like: 
`profilePictureImageFilename: absolutePath + 'Jordan_profile_image.jpg'` ?

Reply:
No. The constant should be just the filename, i.e. 'Jordan_profile_image.jpg'. The code that makes use of this constant should append the correct prefix.

Reply:
Ah. So this would be done at VolunteerProfilesService then. Let me push my changes and see if it is what you want to see :)